config/serverless.yml

```diff
@@ -13,6 +13,8 @@ xpack.cloud.base_url: "https://cloud.elastic.co"
 xpack.cloud.profile_url: "/user/settings"
 xpack.cloud.billing_url: "/billing"
 xpack.cloud.organization_url: "/account"
+xpack.cloud.performance_url: "/performance/"
+xpack.cloud.users_and_roles_url: "/users-and-roles/"
```

Comment:
__question:__ any reason you use trailing slash for `performance_url` and `users_and_roles_url` and **don't** use it for `profile_url`, `billing_url`, and `organization_url` (e.g. you dynamically append more segments to some URLs, but not to the others)? How does the Cloud proxy handle trailing slashes ([redirect or rewrite or something else](https://github.com/slorber/trailing-slash-guide))?

If there is no particular reason, let's try to be consistent and pick a single approach that works the best with the Cloud proxy.